exercises/practice/word-count/tests/word_count.cairo

```diff
@@ -0,0 +1,179 @@
+use super::count_words;
+
+#[test]
+#[ignore]
```

Comment:
Read https://github.com/exercism/cairo/pull/304/files#r1835693572 carefully:

> All tests _except_ the first should have `#[ignore]` attribute, see other exercise tests

The first test is the only one that should _not_ have this attribute, all the subsequent ones should have it.